src/opm/parser/eclipse/EclipseState/Schedule/Group/Group2.cpp

```diff
@@ -195,7 +180,7 @@ bool Group2::wellgroup() const {
 
 bool Group2::addWell(const std::string& well_name) {
     if (!this->m_groups.empty())
-        throw std::logic_error("Groups can not mix group and well children. Trying to add well: " + well_name + " to group: " + this->name());
+        throw std::logic_error("Groups can not mix group and well children");
```

Comment:
Just out of curiosity, why do you remove useful information here?

Reply:
Mistake